cmd/operator-sdk/bundle/validate.go

```diff
@@ -143,22 +168,38 @@ func (c bundleValidateCmd) run() (err error) {
 		logger.Info("Unpacked image layers")
 		err = val.PullBundleImage(c.imageTag, dir)
 		if err != nil {
-			logger.Fatalf("Error to unpacking image: %v", err)
+			msg := "Error to unpacking image"
+			if c.json {
+				log.Fatalf("\nResult:\n%s\n", newOutPut(msg, newErrorList(err)).toJSON())
+			}
+			log.Fatal(msg+": %v", err)
 		}
 	}
 
 	// Validate bundle format.
 	if err = val.ValidateBundleFormat(dir); err != nil {
```

Comment:
You can cast this to a [`ValidationError`](https://github.com/operator-framework/operator-registry/blob/c9610b02f7f4865d7f00fd2b9438a119e2059d32/pkg/lib/bundle/errors.go#L10) to access the underlying errors.